code/src/components/News/Album/Album.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const Album = (props) => {
+  const artistsList = props.artistName.map((element) => {
+    return (<span>{element.name} </span>)
+  })
```

Comment:
Nitpicking, but this would be nicer as an implicit return. There's nothing wrong with this approach (preparing the elements outside of render and then just using the variable further down), but in larger apps this can make things harder to follow. I'd push for doing the .map within the returning JSX as much as possible, until you hit a point where pulling it up and out is warranted.